README.md

```diff
@@ -78,8 +78,8 @@ Group meeting times are listed below:
 
 - US:   Weekly on Wednesdays at 10:00am UTC-7 (see your timezone
   [here](https://time.is/1000_today_in_PT?CNCF_Security_TAG_Meeting))
-- APAC: Bi-weekly on Tuesdays at 1:00pm UTC+11 (see your timezone
-  [here](https://time.is/1300_today_in_AEDT?CNCF_Security_TAG_Meeting))
+- APAC: Bi-weekly on Mondays at 7:00pm PDT UTC-7 (see your timezone
+  [here](https://time.is/1900_today_in_AEDT?CNCF_Security_TAG_Meeting))
```

Comment:
I dont think this time is right...  unless you're proposing a change in the APAC meeting time?

Reply:
Also - since this is aimed at APAC members, the timezone probably should not be a US / EU based timezone